.gitattributes

```diff
@@ -89,5 +99,17 @@ access_log text
 *.pdf binary
 *.png binary
 
-
-**/examples/* linguist-vendored
+# All input for parsers are to be ignored. Issue #2667
+**/examples/** linguist-vendored
+**/example-scripts/** linguist-vendored
+**/example-scriptlets/** linguist-vendored
+**/examples-sql-script/** linguist-vendored
+**/examples_todo/** linguist-vendored
+**/example/test_scripts/** linguist-vendored
+agc/notworking/** linguist-vendored
+asm/asmMASM/helloworld2.asm linguist-vendored
+basic/print.bas linguist-vendored
+erlang/src/** linguist-vendored
+solidity/test.sol linguist-vendored
+sql/postgresql/examples.errors/** linguist-vendored
+v/failed_examples/** linguist-vendored
```

Comment:
Is it possible to use common directories structure for all grammars without exclusions? I mean all examples in `examples` dir, failed examples in `examples-failed` dir etc. Without single vendored files.

Reply:
We are trying to move to a common directory structure, but the work is not complete. And, people keep adding stuff that doesn't fit the standard structure we've settled on. That is:

* Place the common grammar, pom.xml, and a readme in the root directory, e.g., "dart2/Dart2Lexer.g4", "dart2/Dart2Parser.g4", "dart2/pom.xml".
* Place target-specific code in a directory named after the target, e.g., "dart2/Java/Dart2LexerBase.java".
* Place examples in an "examples" directory, e.g., "dart2/examples".

I can only work so fast in bringing order into this chaos. But, I was doing it on a per-grammar basis: clean up the chaos for one grammar at a time, submit a PR, then move on to the next grammar. In fact, I was going to clean up the mess in the python/ directory next. I was in the middle of that when I saw someone create PR "some updates" (with no description, and no Issue created), which only perpetuated the chaos.

I suppose we can tighten up the .gitattribute pattern in some cases, if Linguist globbing works as one would expect:
```
**/examples/** linguist-vendored
**/example-scripts/** linguist-vendored
**/example-scriptlets/** linguist-vendored
**/examples-sql-script/** linguist-vendored
**/examples_todo/** linguist-vendored
**/example/test_scripts/** linguist-vendored
==>
**/example*/** linguist-vendored
```

And, it seems we still don't have a common structure. "examples-failed"?? There's no such thing. We have "examples/" directories, and that may contain .errors files for tests that are expected to fail.

Reply:
Ok, you suggest merging for now? We can fix the rest issues later?

Reply:
Yes, please. I know it's kind of a mess, but I really wonder if github is going to display the new language stats, and I'll rememberto put on my to do list when reorganizing more. I kind of don't trust anything unless I actually see it work, but do see it locally using the tool from the command line. Lots to do. Ty.